modules/standard/FileSystem.chpl

```diff
@@ -509,14 +509,20 @@ proc exists(name: string): bool {
 }
 
 
-/* iter findfiles(startdir = ".", recursive=false, hidden=false)
+/* Finds files from a given start directory and yields their names,
+   similar to simple invocations of the command-line `find` utility.
+   May be invoked in serial or non-zippered parallel contexts.
 
-   findfiles() is a simple find-like utility implemented using the
-   above routines
+   :arg startdir: the root directory from which to start the search, defaults to "."
```

Comment:
The display for "." is a little weird.  I wonder if surrounding it with backticks will improve it?